packages/amplify-e2e-tests/src/__tests__/function_1.test.ts

```diff
@@ -465,6 +465,21 @@ describe('nodejs', () => {
         path.join(projRoot, 'amplify', 'backend', 'function', fnName, `${fnName}-cloudformation-template.json`),
       );
       expect(lambdaCFN.Resources.AmplifyResourcesPolicy.Properties.PolicyDocument.Statement.length).toBe(3);
+
+      const configPath = path.join(projRoot, 'amplify', 'backend', 'backend-config.json');
```

Comment:
I'd move this check to it's own test. It's confusing to have this in a test titled "adding api and storage permissions should not add duplicates to CFN"